lib/mruby/build/load_gems.rb

```diff
@@ -68,32 +68,32 @@ def initialize(gemdir, repo, branch, commit, canonical, path = nil)
 
       def full_gemdir
         return @gemdir unless @path
-        return File.join(@gemdir, @path)
+        File.join(@gemdir, @path)
       end
 
-      def canonical?()  return @canonical;                  end
-      def git?()        return !!@repo;                     end
-      def gemname()     return File.basename(@gemdir);      end
+      def canonical?()  @canonical;                  end
+      def git?()        !!@repo;                     end
+      def gemname()     File.basename(@gemdir);      end
 
       def hash
-        return [@gemdir, @repo, @branch, @commit, @canonical, @path].hash
+        [@gemdir, @repo, @branch, @commit, @canonical, @path].hash
       end
 
       def ==(other)
-        return @gemdir == other.gemdir &&
-          @repo == other.repo &&
-          @branch == other.branch &&
-          @commit == other.commit &&
-          @canonical == other.canonical? &&
-          full_gemdir == other.full_gemdir
+        @gemdir == other.gemdir &&
+        @repo == other.repo &&
+        @branch == other.branch &&
+        @commit == other.commit &&
+        @canonical == other.canonical? &&
+        full_gemdir == other.full_gemdir
```

Comment:
`return` is not important. However, the indentation here is very ( :smile_cat: ) important, so could you keep it?

Reply:
Thanks @dearblue the indentation is now fixed